coffee/session.coffee

```diff
@@ -124,12 +133,18 @@ class palava.Session extends @EventEmitter
     @room.on 'signaling_not_reachable', (p) => @emit 'signaling_not_reachable', p
     true
 
+  # Reconnect the session
+  #
+  reconnect: =>
+    @destroy(false)
```

Comment:
It releasing the user media necessary -- maybe we should create a new destroy method?

Reply:
Same goes for getting user media in init(). If we'd have a second method there, too, it would make argument handling easier, too.

Reply:
You're correct, usermedia should not be rerequested - what do you think about the change that I made?

Reply:
:+1: